dotnet/samples/Connectors/EventDrivenTcpThermostatConnector/KubernetesResources/connector-template.yaml

```diff
@@ -13,9 +13,11 @@ spec:
     managedConfigurationSettings:
       managedConfigurationType: imageConfiguration
       imageConfigurationSettings:
-        image: eventdriventcpthermostatconnector:latest
+        imageName: "eventdriventcpthermostatconnector"
         imagePullPolicy: Never # This sample image is built locally, so no need to pull
-        replicas: 1
+        tagDigestSettings:
+          tagDigestType: tag
+          tag: "latest"
```

Comment:
The shape of these yaml files changed from the operator side recently, so I'm updating them in our samples accordingly